ocs_ci/ocs/perf_pgsql.py

```diff
@@ -0,0 +1,87 @@
+"""
+PGSQL performance workload class
+"""
+import logging
+
+from prettytable import PrettyTable
+
+from ocs_ci.ocs.pgsql import Postgresql
+
+log = logging.getLogger(__name__)
+
+
+class PerfPGSQL(Postgresql):
```

Comment:
why do we need a new class for creating the benchmark operator and not using the existing class?

Reply:
> why do we need a new class for creating the benchmark operator and not using the existing class?

@keesturam We can use this in the need of furthermore validations specific to performance. For eg:-  We have a function called "validate_pgbench_perf" in this class which prints only the required data. Similarly, we may have such scenarios in the future. So thought of having this like a customizing class.